package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ojs-inline-components",
-  "version": "1.1.0",
+  "version": "1.4.0",
```

Comment:
Bo jest to trzeci komponent z kolei, który będzie mergowany. Dlatego podbicie aż o 3 minory